app/assets/stylesheets/application.css.scss

```diff
@@ -6,3 +6,16 @@
 @import "daplog/base";
 @import "daplog/blocks";
 @import "daplog/contexts";
+
+#foo{}
+a.Bar   {
+    margin : 0 0 0;
```

Comment:
Rule sets should be ordered as follows: @extends, @includes without @content, properties, @includes with @content, nested rule sets<br>Line should be indented 2 spaces, but was indented 4 spaces<br>Properties should be ordered backgrounds, border, border, -webkit-border-radius, content, margin<br>Shorthand form for property `margin` should be written more concisely as `0` instead of `0 0 0`<br>Property name should be immediately followed by a colon